src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -1169,53 +1171,63 @@ const FormCreationForm = ({formUuid, formUrl, formHistoryUrl}) => {
             <Tab hasErrors={state.tabsWithErrors.includes('form')}>
               <FormattedMessage defaultMessage="Form" description="Form fields tab title" />
             </Tab>
-            <Tab hasErrors={state.tabsWithErrors.includes('form-steps')}>
-              <FormattedMessage
-                defaultMessage="Steps and fields"
-                description="Form design tab title"
-              />
-            </Tab>
+            {!isAppointment && (
```

Comment:
Why is this condition needed? I would expect that since that an appointment form doesn't have steps/variables/registration backend, it wouldn't have errors?

Reply:
This is hiding the tab for steps and fields, not the "non form field"-errors. Or am I confused here?

Reply:
Ohhh my bad, I understand now.